src/visitors/Tag.js

```diff
@@ -35,22 +38,38 @@ function getAttributes(node: Object, context: Context): Array<JSXAttribute | JSX
           name = 'className';
           break;
       }
-      const expr = parseExpression(val === true ? 'true' : val, context);
-      if (!mustEscape && (!t.isStringLiteral(expr) || /(\<\>\&)/.test(val))) {
-        throw new Error('Unescaped attributes are not supported in react-pug');
+
+      let expr;
+
+      if (getInterpolationRefs(val)) {
```

Comment:
Can you move this code into `parseExpression` too, that way it will be accessible to everywhere we attempt to parse an expression.  e.g. you would be able to do:

```pug
while (${foo})
  div
```